scripts/wrist_test.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+
+import rospy
+from sensor_msgs.msg import JointState
+import numpy as np
+
+rospy.init_node("wrist_test")
+topic_root = '/roboy/pinky/'
+publisher = rospy.Publisher("/roboy/pinky/control/joint_targets", JointState, queue_size=1)
+rate = rospy.Rate(100)
+joints_name = ['shoulder_left_axis0','shoulder_left_axis1','shoulder_left_axis2',
+               'elbow_left_axis0', 'elbow_left_axis1',
+               'wrist_left_axis0', 'wrist_left_axis1', 'wrist_left_axis2',
+               'head_axis0', 'head_axis1', 'head_axis2',
+               'shoulder_right_axis0', 'shoulder_right_axis1', 'shoulder_right_axis2',
+               'elbow_right_axis0', 'elbow_right_axis1',
+               'wrist_right_axis0', 'wrist_right_axis1', 'wrist_right_axis2']
+
+test_lr = "right"
+notest_lr = "left" if test_lr == "right" else "right"
+
+
+def hard_reset():
+    msg = JointState()
+
+    msg.name = joints_name
+    msg.position = [0]*len(msg.name)
+    msg.velocity = [0]*len(msg.name)
+    msg.effort = [0]*len(msg.name)
+    publisher.publish(msg)
+
+    rospy.loginfo("Hard reset")
+    rospy.sleep(2.0)
+
+
+def soft_reset():
+    pass
+
+
+def hand_shake(lr="right"):
+    msg = JointState()
+    msg.name = ["elbow_"+lr+"_axis0"]
+    msg.velocity = [0]*len(msg.name)
+    msg.effort = [0]*len(msg.name)
+    ranges = [0.6, 1.0]
+    samples = 50
+    for i in range(10):
+        if not i % 2:
+            rospy.loginfo("up")
```

Comment:
add meaningful logging

Reply:
Removed redundant logs and added more informative ones